Documentation/TypoScriptConfiguration/Index.rst

```diff
@@ -199,6 +199,9 @@ Therefore, :ts:`.external = 1` forces TYPO3, not to check for their local
 existence. The fourth JavaScript file is the file we added before to the site
 package extension itself.
 
+You can also include CSS or JavaScript per-component in your Fluid template or 
+by PHP. See :ref:`Assets <t3coreapi:assets>`.
```

Comment:
What about writing :ref:`t3coreapi:assets`? That would use the text of the title of that chapter as linktext.

Reply:
Fine with me of course. The title looked a bit clunky to me in that place https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/Assets/Index.html

Reply:
Changed it.